charts/kyverno/templates/clusterrolebinding.yaml

```diff
@@ -60,6 +60,45 @@ roleRef:
   kind: ClusterRole
   name: {{ template "kyverno.fullname" . }}:generatecontroller
 subjects:
+- kind: ServiceAccount
+  name: {{ template "kyverno.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+---  
+kind: ClusterRoleBinding
```

Comment:
I don't think we want to install Kyverno with admin role acces by default. We should enable the least privileged set required, and allow customers to expand.